Xamarin.Forms.Platform.Android/Renderers/BoxRenderer.cs

```diff
@@ -44,16 +45,47 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 			if (e.PropertyName == BoxView.ColorProperty.PropertyName || e.PropertyName == VisualElement.BackgroundColorProperty.PropertyName)
 				UpdateBackgroundColor();
+			else if (e.PropertyName == BoxView.CornerRadiusProperty.PropertyName)
+				UpdateCornerRadius();
 		}
 
 		protected override void UpdateBackgroundColor()
 		{
 			Color colorToSet = Element.Color;
 
-			if (colorToSet == Color.Default)
-				colorToSet = Element.BackgroundColor;
+			var background = new GradientDrawable();
```

Comment:
Is there a reason we have to create a new GradientDrawable each time we update the BackgroundColor? Can we just keep the same one around and update its properties?

Reply:
Without a doubt, is better to create one and update. In fact it is done in many other Renderers already. Changed!. Another point, I add the use of Dispose to release the GradientDrawable. What do you think?